assets/js/components/common/DebugEntry.jsx

```diff
@@ -196,7 +196,7 @@ class DebugEntry extends Component {
             )
           }
           {
-            event.data.integration && (
+            this.state.showIntegrationInfo && (
```

Comment:
you can't show integration info if event.data.integration doesnt exist right?

Reply:
you right, otherwise it'd crap out again, added condition